doc/release/release_0.14.rst

```diff
@@ -1,3 +1,66 @@
+Announcement: scikit-image 0.14.3
+=================================
+
+This is a bugfix release, and contains the following changes from v0.14.2:
+
+API Changes
+-----------
+-  morphology.local_maxima now returns a boolean array instead of uint8 (#3749,
+   #3752)
+
+
+Bug Fixes
+---------
+- _marching_cubes_lewiner_cy: mark char as signed (#3587, #3678)
+- Fix potential use of NULL pointer (#3696)
+- pypi: explicitly exclude Python 3.1, 3.2, and 3.3 (#3726)
+- Reduce default tolerance in threshold_li (#3622) (#3781)
+
+Other Pull Requests
+-------------------
+
+- BLD: pin cython's language_level (#3716)
+- Build tools: Upgrade xcode to 9.4 on v0.14.x branch (#3724)
+- Get rid of the requirements-parser dependency (#3534, #3727)
+- Add small galleries in the API (#2940, #3728)
+- Correctly ignore release notes auto-generated for docs (#3656, #3737)
+- Fix qt 5.12 for 0.14. (#3744, #3753)
+- Set astropy minimum requirement to 1.2 to help the CIs. (#3767, #3770)
+- Avoid NumPy warning while stacking arrays. (#3768, #3771)
+- Fix human readable error message on a bad build. (#3223, #3790)
+- Unify LICENSE files for easier interpretation (#3791, #3792)
+
+
+11 authors added to this release [alphabetical by last name]
+------------------------------------------------------------
+- François Boulogne
+- Thomas A Caswell
+- Christoph Gohlke
+- Lars Grueter
+- Mark Harfouche
+- Juan Nunez-Iglesias
+- Egor Panfilov
+- Nelle Varoquaux
+- Stefan van der Walt
+- Nehal J Wani
+
+... and, as always, a special mention to Matthias Bussonnier's Meeseeks Box,
+which remains invaluable for our backports.
+
+3 committers added to this release [alphabetical by last name]
+--------------------------------------------------------------
+- Mark Harfouche
+- Stefan van der Walt
+- Josh Warner
```

Comment:
```suggestion
- Josh Warner
- AetherUnbound
```

Reply:
AetherUnbound -> Matthew Bowden

Reply:
Whats the difference between a committer and an author?